jobs/e2e-test.groovy

```diff
@@ -288,8 +301,18 @@ onWorker(WORKER_TYPE, '5h') {     // timeout
            buildmaster: [sha: params.GIT_REVISION,
                          what: E2E_RUN_TYPE]]) {
       initializeGlobals();
+      stage("Generate skipped list") {
+         dir("webapp/services/static") {
+            sh("pnpm cypress:clean");
+            exec(["./dev/cypress/e2e/tools/gen-skipped-e2e-tests.js", "${E2E_URL}"]);
+            stash includes: "skipped_e2e_tests.json", name: "e2e-skipped-list";
+         };
+      }
       stage("Run e2e tests") {
-         withEnv(["COMMIT_INFO_BRANCH=${SHORT_REVISION_DESCRIPTION}"]) {
+         withEnv([
+            "COMMIT_INFO_BRANCH=${SHORT_REVISION_DESCRIPTION}",
+            "KA_SKIP_GEN=1",
```

Comment:
Side note: KA_SKIP_GEN is not a very descriptive name.  I had to look at the source to see what it does.  I'd suggest renaming it -- in some other PR -- to KA_DO_NOT_GENERATE_SKIPPED_TESTS or some other long but descriptive name.

Reply:
`KA_SKIP_GEN` was used to skip various "generation" steps - not just the generation of skipped tests, but also of some other files...I think some have since gone away.

Reply:
No, the files that were skipped before were appropriate in this case too. That's not a concern.

Reply:
The name can definitely be better though; but that's a change for a different PR, as you say. And one that needs to be done carefully since it's cross-repo.